charts/admission-controller/values.yaml

```diff
@@ -223,11 +256,24 @@ webhook:
       # <br/>And inbound connections to serve HttpRequests as Kubernetes Webhook.
       # <br/>A PEM-encoded x509 certificate authority.
       # <br/>If empty, a new CA will be autogenerated.
-      cert: ""
+      cert:
```

Comment:
@aroberts87 @mavimo 
Using the `webhook.ssl.ca.cert` value for the scope of this PR might actually cause a BC.
This is currently used in https://github.com/sysdiglabs/charts/blob/master/charts/admission-controller/templates/_helpers.tpl#L166 in order to generate a cert that can be later used in https://github.com/sysdiglabs/charts/blob/master/charts/admission-controller/templates/webhook/admissionregistration.yaml#L42
Since `global.ssl.ca.cert` and `<component>.ssl.ca.cert` actually contains more than one cert, it might actually make sense to rename it in `certs` (actually `rapid-response` and `registry-scanner` already use `certs` for this purpose).
WDYT?

Reply:
`certs` LGTM (also because it can contains multiple certs 😅 )
Are you considering to make it an array or just a string with multiple certs?

Reply:
@AlbertoBarba @mavimo @aroberts87 

The reason I didn't touch the local CA on these components was they were written before the global.  now, technically a user should never need more than 1 root CA to add, however, you can bundle them into 1 file per SSL RFC.  So this is why I opted to just do a string in the global section as they can concatenate as many as they want.

I couldn't change the local's on these because we don't know if any customers are using them to break their deployments.

Just my last 2c while I'm here.  :)

Reply:
@dthomson-sysdig exactly...as u said we are not going to change the local CA of these components. In order to keep it in place and avoid any possible conflict with the new config we are just going to differentiate the values names. Even if we keep it as a string I'm not 100% sure that https://github.com/sysdiglabs/charts/blob/master/charts/admission-controller/templates/_helpers.tpl#L166 will work properly if a customer specifies more than one CA. I guess we'll have to figure out how to solve it once we'll decide to make a major release and do a BC.